### PR TITLE
es6-template-literals - Susana Gutierrez

### PR DESCRIPTION
**Pull request by susieggrim**
es6-teplate-literals

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is  ${firstName}  ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: @Susana-Gutierrez You missed a couple concatenations

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by susieggrim** [comment]
es6-teplate-literals2

File: es6-template-literals/index.html (hunk 1 above)
Comment (susieggrim) [reply es6-template-literals/index.html]: Missed concatenations changed.

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Looks like an extra space was added here:

---

image

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit}${item}`;
+
+    console.log(`I have a ${item}`);
+    console.log(`I have an ${fruit}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Passing `console.log` a single string template argument is not the same operation as passing it two arguments:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by susieggrim** [comment]
es6-teplate-literals3

File: es6-template-literals/index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    console.log('LearningFuze:', address);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: One of the console logs is still different from its original version:

---

image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**